From 5e76e10095d6370a3c7235543acda9b4f256d246 Mon Sep 17 00:00:00 2001
From: Tomas Novotny <tomas@novotny.cz>
Date: Tue, 24 Oct 2017 15:07:42 +0200
Subject: [PATCH 27/82] arm: dts: sun8i: a83t: a711: update brightness levels
 to fit the device

The function is exponential with base of 1.32 and some offset and
tweaks. The lowest values produce no output, so we are starting at 8.
The default brigtness is a bit lower than maximum to boot with reduced
backlight.

Signed-off-by: Tomas Novotny <tomas@novotny.cz>
---
 arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
index dc7b94a6c068..bd7e231e3aba 100644
--- a/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
+++ b/arch/arm/boot/dts/sun8i-a83t-tbs-a711.dts
@@ -65,8 +65,11 @@
 		pwms = <&pwm 0 50000 PWM_POLARITY_INVERTED>;
 		enable-gpios = <&pio 3 29 GPIO_ACTIVE_HIGH>;
 
-		brightness-levels = <0 1 2 4 8 16 32 64 128 255>;
-		default-brightness-level = <9>;
+		brightness-levels = <0 8 9 10 11 13 15 17 19 21 23 25 28
+				     31 34 37 40 44 48 52 56 61 67 72 78
+				     84 91 98 106 115 124 133 143 154
+				     166 179 192 207 223 239 255>;
+		default-brightness-level = <39>;
 	};
 
 	panel {
-- 
2.20.1

